locales/de.json

```diff
@@ -862,145 +862,145 @@
+  "app.privacy.cookies.storeInfo": "Wir nutzen Cookies, um Informationen für Ihren Komfort zu speichern (“Einstellungen“). Wie können ebenso Drittparteien-Werbung auf dieser Seite verwenden. Manche von diesen Werbenden können Technologien wie Cookies und Web Beacons verwenden, wenn sie auf dieser Seite Werbung betreiben, was diesen Werbenden (so wie Google durch das Google AdSense Programm) Informationen, welche Ihre IP-Adresse, Ihren Internetanbieter, den Browser, mit dem Sie diese Seite aufgerufen haben und in manchen Fällen, ob Sie Flash installiert haben, einschließen, senden werden. Dies wird generall für den Zweck des Geotargetings getan (beispielsweise um jemandem in New York Immobilien-Angebote aus New York zu zeigen) oder um gewisse Werbungen basierend auf bestimmten besuchten Seiten anzuzeigen (so wie Koch-Werbungen für jemanden, der regelmäßig Kock-Seiten besucht).",
+  "app.privacy.dataPortability.exercise": "Um die oben beschriebenen Rechte zu Zugriff, Datenauskunft und Löschung auszuüben, stelle Sie eine verifizierbare Verbraucheranfrage an uns, indem Sie uns eine Email sendest an",
+  "app.privacy.dataPortability.registered": "Nur Sie, oder eine Person, die beim Kalifornischen Staatssekretär dafür registriert ist, in Ihrem Namen handeln zu dürfen, dürfen eine verifizierbare Verbraucheranfrage in Bezug auf deine persönlichen Informationen stellen. Sie dürfen auch eine verifizierbare Verbraucheranfrage im Namen Ihres minderjährigen Kindes stellen.",
+  "app.privacy.dataPortability.respond": "Wir können nicht auf Ihre Anfrage reagieren oder Sie mit persönlichen Informationen versorgen, falls wir Ihre Identität oder Autorität für die Durchführung der Anfrage nicht verifizieren können und wenn wir nicht bestätigen können, dass sich die persönlichen Informationen auf Sie beziehen.",
+  "app.privacy.dataPortability.title": "Ausübung von Rechten zu Zugriff, Datenauskunft und Löschung",
+  "app.privacy.dataPortability.verifiableConsumer": "Sie können eine verifizierbare Verbraucheranfrage für Datenauskunft nur zwei mal in einer 12-monatigen Periode stellen. Die verifizierbare Verbraucheranfrage muss:",
+  "app.privacy.dataPortability.verifiableConsumer.detail": "Beschreiben Sie Ihre Anfrage mit ausreichenden Details, welche uns erlauben sie richtig zu verstehen, evaluieren und auf sie zu reagieren.",
+  "app.privacy.dataPortability.verifiableConsumer.provideInfo": "Stellen Sie ausreichende Informationen zur Verfügung, welche uns in einem angemessenem Rahmen erlauben zu verifizieren, dass Sie die Person sind, über die wir persönliche Informationen erfasst haben oder ein autorisierter Repräsentant.",
+  "app.privacy.dataPortability.verifiableRequest": "Eine verfizierbare Verbraucheranfrage zu stellen erfordert nicht ein Benutzerprofil bei uns zu erstellen. Wir werden nur persönliche Informationen aus einer verifizierbaren Verbraucheranfrage verwenden, um die Identität oder Autorität des Antragstellers zu verifizieren.",
```

Comment:
"Autorität" is not the correct translation for this type of authority. "Behörde"/"Obrigkeit" might fit better, although it still feels stiff and unnatural.